server/src/test/java/org/elasticsearch/cluster/coordination/ElasticsearchNodeCommandIT.java

```diff
@@ -250,8 +250,7 @@ public void testDetachAbortedByUser() throws IOException {
         expectThrows(() -> detachCluster(environment, 0, true), ElasticsearchNodeCommand.ABORTED_BY_USER_MSG);
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/39220")
-    public void test3MasterNodes2Failed() throws Exception {
+    public void test3MasterNodeRs2Failed() throws Exception {
```

Comment:
Renaming this test looks like it wasn't intended.

Reply:
ah sorry, didn't see that, thx!